sites/page-details/index.json-ld

```diff
@@ -0,0 +1,86 @@
+{
+	"@context": {
+		"@version": 2.0,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Page details",
+		"fr": "Détails de la page"
+	},
+	"description": {
+		"en": "Documentation on how to use the page details section.",
+		"fr": "Documentation sur l'utilisation de la section des détails de la page."
+	},
+	"modified": "2022-04-11",
+	"componentName": "page-details",
+	"status": "stable",
+	"pages": {
+		"docs": [
+			{
+				"title": "Documentation for Page details",
+				"language": "en",
+				"path": "page-details-docs-en.html"
+			},
+			{
+				"title": "Documentation pour Détails de la page",
+				"language": "fr",
+				"path": "page-details-docs-fr.html"
+			}
+		],
+		"examples": [
+			{
+				"title": "Default page details",
+				"language": "en",
+				"path": "default-page-details-en.html"
+			},
+			{
+				"title": "Détails de la page par default",
+				"language": "fr",
+				"path": "default-page-details-fr.html"
+			},
+			{
+				"title": "Page details without 'Report a problem' and 'Share this page' buttons",
+				"language": "en",
+				"path": "page-details-without-buttons-en.html"
+			},
+			{
+				"title": "Détails de la page sans les boutons 'Signaler un problème' et 'Partager cette page'",
+				"language": "fr",
+				"path": "page-details-without-buttons-fr.html"
+			},
+			{
+				"title": "Page details without 'Report a problem' button",
+				"language": "en",
+				"path": "page-details-without-report-btn-en.html"
+			},
+			{
+				"title": "Détails de la page sans le bouton 'Signaler un problème'",
+				"language": "fr",
+				"path": "page-details-without-report-btn-fr.html"
+			},
+			{
+				"title": "Page details without 'Share this page' button",
+				"language": "en",
+				"path": "page-details-without-share-btn-en.html"
+			},
+			{
+				"title": "Détails de la page sans le bouton 'Partager cette page'",
+				"language": "fr",
+				"path": "page-details-without-share-btn-fr.html"
+			},
+			{
+				"title": "Page details v.1.0 (deprecated)",
+				"language": "en",
+				"path": "deprecated/page-details-v1-en.html"
+			},
+			{
+				"title": "Détails de la page version 1.0 (obsolète)",
+				"language": "fr",
+				"path": "deprecated/page-details-v1-fr.html"
+			}
```

Comment:
Info.
We should create a different category in the page for the deprecated page that we still support.